### PR TITLE
Fix for call to removed function

### PR DESCRIPTION
**Pull request by akinsey**
Please test that the server and app run properly

### CODE REVIEW

**Comment by unenglishable** [comment]
image

File: modules/ept-frontend/client/app.js [hunk 1]
```diff
@@ -75,10 +75,7 @@ require('./patrol/resource');
 // Set Angular Configs
 app
 .config(require('./config'))
-.run(['$rootScope', '$state', '$timeout', 'Alert', 'BreadcrumbSvc', 'BanSvc', 'Websocket', function($rootScope, $state, $timeout, Alert, BreadcrumbSvc, BanSvc, Websocket) {
-
-  // start the websocket connection on page load
-  Websocket.init();
```
Comment (unenglishable) [inline modules/ept-frontend/client/app.js]: are you sure this is the only place it is called?

Comment (akinsey) [reply modules/ept-frontend/client/app.js]: I think so.

Comment (unenglishable) [reply modules/ept-frontend/client/app.js]: can you please be certain?  i can't approve of this change unless you can say for sure.

Comment (unenglishable) [reply modules/ept-frontend/client/app.js]: can you please be certain? i can't approve of this change unless you can say for sure.

Comment (akinsey) [reply modules/ept-frontend/client/app.js]: it is

Comment (unenglishable) [reply modules/ept-frontend/client/app.js]: okay

**Pull request merged**